devel/ld64/files/PR-30596832.patch

```diff
@@ -0,0 +1,11 @@
+--- libtapi/lib/Core/Symbol.cpp.orig	2017-01-09 13:47:49.000000000 -0800
++++ libtapi/lib/Core/Symbol.cpp	2017-02-19 16:46:11.000000000 -0800
+@@ -9,7 +9,7 @@
+ 
+ #include "tapi/Core/Symbol.h"
+ #include "tapi/Core/LLVM.h"
+-#include "llvm/Config/config.h"
++//#include "llvm/Config/config.h"
```

Comment:
Why comment out the `#include`? Why not just remove it? Commenting things out in a patchfile just makes the patchfile unnecessarily larger and more verbose, and ultimately harder to understand.